app/models/application.rb

```diff
@@ -12,6 +12,10 @@ class Application < ActiveRecord::Base # rubocop:disable ClassLength
     where(benefits: false, application_type: 'income', application_outcome: %w[part full])
   }
 
+  scope :waiting_for_evidence, lambda {
+    includes(:evidence_check).references(:evidence_check).where.not(evidence_checks: { id: nil })
+  }
+
```

Comment:
I agree that the model is fat at the moment (work in progress), but scopes are the things that belong here, imho. Lot of the other stuff doesn't. 

I'm not against moving this to a query model, however I'd wait until we finish the evidence check flow and then know what these queries actually look like. I wasn't even sure if this is a concern of `Application` or `EvidenceCheck`. But `EvidenceCheck` does not belong to `Office`, therefore I chose to put it here.